src/style.css

```diff
@@ -1,3 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
```

Comment:
do not reset margin for all selectors. only for elements where it is necessary.
generally * selector only for box-sizing property